src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -933,6 +935,7 @@ public List<Setting<?>> getSettings() {
             settings.add(Setting.boolSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_RESOLVE_INDICES, true, Property.NodeScope, Property.Filtered));
             settings.add(Setting.boolSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_REST, true, Property.NodeScope, Property.Filtered));
             settings.add(Setting.boolSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_TRANSPORT, true, Property.NodeScope, Property.Filtered));
+            settings.add(Setting.boolSetting("plugins.security.audit.config.enable_transport", true, Property.NodeScope, Property.Filtered));
```

Comment:
`"plugins.security.audit.config.enable_transport"` should it be moved to ConfigConstants?

Reply:
Good catch, this is covered as part of the section of code at lines 950